docs/build/js-client/3-setting-up.md

```diff
@@ -46,23 +44,35 @@ Create `tsconfig.json` file and add following content there
     "strict": true,
     "skipLibCheck": true,
     "moduleResolution": "nodenext"
-  }
+  },
+  "include": ["src"]
 }
 ```
 
+Create a new file called `fluence.yaml` and put there following content:
```

Comment:
maybe instead of that suggest people to init new project using `fluence init` - this file will be created for them